public/views/psychologistForm.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MindSet</title>
+    <link rel="stylesheet" href="../styles/reset.css">
+    <link rel="stylesheet" href="../styles/styles.css">
+    <link rel="stylesheet" href="../styles/psychologist.css">
+    <script async src="../js/psychologistForm.js"></script>
+  </head>
+  <body>
+    <header>
+      <a id="mindSetLogo" href="/"><span>Mind</span>SET</a>
+      <nav>
+        <a href="#">admins</a>
+        <a href="#">applications</a>
+        <a href="#">clients</a>
+        <a href="#">interviews</a>
+        <a href="#">positions</a>
+        <a href="#">Profiles</a>
+        <a id="psychologistNav" href="./psychologistForm.html">psychologists</a>
+        <a href="#">sessions</a>
+      </nav>
+    </header>
+    <div class="container">
+      <h2 class="header">Psychologist Form</h2>
+      <form id="form">
+        <div>
+          <input id="firstName" name="firstName" placeholder="Psychologist First Name" type="text" required pattern="[A-Za-z ]*" title='Enter a valid first name'"/>
```

Comment:
try to add some margin to separate each input from the others

Reply:
Fixed it.